01_Core/week03/submissions/exercise 1/index.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">	
+  <title>CSS Exercise 1</title>
+  <meta name="description" content="ITA CSS Layout Exercise">
+  <meta name="author" content="Mark Kroh">
+
+  <style>
+	  .container  {
+		  width: 275px;
+		  height: 275px;
+	  }
+
+	  .interior	{
+		  width: 100px;
+		  height: 100px;
+		  margin: 10px 10px;
```

Comment:
If it's the same margin all the way around you can actually just do `margin: 10px;`